subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/tasks/ValidateTaskPropertiesIntegrationTest.groovy

```diff
@@ -54,9 +49,8 @@ class ValidateTaskPropertiesIntegrationTest extends AbstractIntegrationSpec {
                     return count;
                 }
 
-                // Ignored because not a JavaBean getter
                 public long getter() {
```

Comment:
How about we remove the `getter` method completely? It doesn't give any new information in this test, does it?

Reply:
I think it's good information that we detect this as a property called `ter`.